pipeline-steps/genesis.yaml

```diff
@@ -4,72 +4,26 @@ parameters:
   serviceConnection: ''
   stack: ''
   project: ''
-  tfversion: ''
-  tfInitSub: ''
-  builtFrom: ''
   product: ''
+  tfInitSub: ''
```

Comment:
is this file (genesis.yaml) actually required anymore?

i.e what's stopping it from using the `steps/terraform.yaml@cnp-azuredevops-libraries` ?

Reply:
There is just one extra step in genesis.yaml (pasted below) before teraform plan and apply. Maybe it could be moved to the main pipeline?

`- script: rm -fr .terraform/modules/${{ parameters.stack }}`

Reply:
Note: the genesis.yaml file already uses steps/terraform.yaml@cnp-azuredevops-libraries, just has the extra step pasted in previous comment

Reply:
pretty sure not needed, no one has touched that file in a long time i would just remove it and see if any issues

Reply:
Done. Removed `genesis.yaml` and used `steps/terraform.yaml@cnp-azuredevops-libraries` directly without the extra step mentioned above. No issues in pipeline